packages/code-du-travail-modeles/src/modeles/conventions/2511_sport/indemnite-precarite.yaml

```
@@ -0,0 +1,54 @@
+
+          
+          
+contrat salarié . convention collective . sport . type de cdd CDD d'usage appelé contrat d'intervention:
+  applicable si: type de cdd = 'CDD d'usage appelé contrat d'intervention'
+          
+          
+contrat salarié . convention collective . sport . type de cdd CDD d'usage appelé contrat d'intervention . embauché en cdi non:
+  applicable si: embauché en cdi = 'non'
+          
+          
+          
+contrat salarié . convention collective . sport . type de cdd CDD d'usage appelé contrat d'intervention . embauché en cdi non . résultat conventionnel:
+  valeur: 
+    produit:
+      assiette: contrat salarié . salaire de référence
+      facteur: 0.1
+  arrondi: 2 décimales
+  unité: €
+  cdtn:
+    formule:
+      explanations:
+        - "Sref : Salaire de référence": contrat salarié . salaire de référence
+      formula: 1/10 * Sref
+  remplace: contrat salarié . convention collective . résultat conventionnel
+  références:
+    "article 4.7 de la convention collective (point 4.7.2.)": https://www.legifrance.gouv.fr/affichIDCCArticle.do;jsessionid=AC096B3D0CD017996E22DEE5D3C1ABFF.tplgfr41s_1?idArticle=KALIARTI000021063900&cidTexte=KALITEXT000017577657&dateTexte=20160726&categorieLien=id
```

```suggestion
  références:
    "Article 4.7 de la convention collective (point 4.7.2.)": https://www.legifrance.gouv.fr/affichIDCCArticle.do;jsessionid=AC096B3D0CD017996E22DEE5D3C1ABFF.tplgfr41s_1?idArticle=KALIARTI000021063900&cidTexte=KALITEXT000017577657&dateTexte=20160726&categorieLien=id
```

et pour en dessous aussi